clients/benchmarks/CMakeLists.txt

```diff
@@ -77,6 +77,7 @@ target_include_directories( client
     $<BUILD_INTERFACE:${Boost_INCLUDE_DIRS}>
 )
 
+link_libraries( ${BOOST_ROOT}/lib )
```

Comment:
NO, It does not give absolutely paths.  If invoking the executable, it fails. 

/home/tim/usr/cget/pkg/kitware__cmake/install/bin/cmake -E cmake_link_script CMakeFiles/client.dir/link.txt --verbose=1
/opt/rocm/hip/bin/hipcc    -m64 -O3 -DNDEBUG   CMakeFiles/client.dir/client.cpp.o CMakeFiles/client.dir/home/tim/rocBLAS_tindong/clients/common/utility.cpp.o CMakeFiles/client.dir/home/tim/rocBLAS_tindong/clients/common/cblas_interface.cpp.o CMakeFiles/client.dir/home/tim/rocBLAS_tindong/clients/common/flops.cpp.o CMakeFiles/client.dir/home/tim/rocBLAS_tindong/clients/common/norm.cpp.o CMakeFiles/client.dir/home/tim/rocBLAS_tindong/clients/common/unit.cpp.o CMakeFiles/client.dir/home/tim/rocBLAS_tindong/clients/common/near.cpp.o CMakeFiles/client.dir/home/tim/rocBLAS_tindong/clients/common/arg_check.cpp.o CMakeFiles/client.dir/home/tim/rocBLAS_tindong/clients/common/rocblas_template_specialization.cpp.o  -o client -rdynamic /usr/local/lib/librocblas-hcc.a **-lboost_program_options** -lblas -llapack -lpthread /usr/local/lib/libtensile-rocblas.a